config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml

```diff
@@ -0,0 +1,64 @@
+# periodic jobs
+
+- name: ci-kubernetes-e2e-kubeadm-kinder-external-etcd-master
+  interval: 2h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 40m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: kubeadm
+    base_ref: master
+    path_alias: k8s.io/kubeadm
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190528-0d7c4b5-master
```

Comment:
the tag might be out of date
i would use https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml#L23

Reply:
Updated.